Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+\label{subsubsec: design phase}
+    The goals are to design the propulsion system, the test stand, the nozzle. At the end of this phase
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
+    will use it as a guide (inlcuding drawings for machinists).
+
+    \textit{Projected Duration: 2--3 weeks}
+
+\subsubsection{Build \& Preliminary Test Phase}
+\label{subsubsec: sanity test phase}
+    This phase is concerned with the building of the system. The test stand is already built. However if minor revisions are necessary, then the team will
+    build the new design (any re-design will concern the dimensions of the tank fixture and will therefore not take long as it is a simplistic system).
+    The main part of the building stage will be the construction of the propulsion system. That is where most of the work will be. Once the system is built,
+    it will be tested at component-level then tested at system-level. This is to ensure there are no catastrophic problems during the experiment.
+    There will be no data collected during this phase.
+
+    \textit{Projected Duration: 3--5 weeks}
+
+\subsubsection{Experimentation}
+\label{subsubsec:experimentation}
+    This phase is primarily concerned with the testing of the CGT\@. This includes gathering all the necessary data and experience to create the
+    polynomial approximation and to aid in the RCS phase and CREST phase of space propulsion research.
+    The testing will consist of 5 tests for each different chamber pressure. These chamber pressures will be; 80, 90, 100, 120, 140, 160, 180, and 200 Psi. There
+    will be a total of 40 tests, if time permits there can be more but this is the goal as it will serve as sufficient data for data analysis.
+     The data will be averaged for each chamber pressure and then plotted against the nozzle area ratio. Then a tenth-order polynomial
+    will be fit to the curve and used to determine a first-order approximation of the nozzle area ratio for future cold gas propulsion systems.
+
+    \textit{Projected Duration: 2--3 weeks}
+
+\subsubsection{Data Analysis}
+\label{subsubsec: data analysis}
+    This phase is concerned with the analysis of the data collected from the CGT testing. The polynomial approximation will be completed during this
+    phase for the minimum length nozzle. It is also the phase where the performance analysis takes place. This will inlcude the ultimate calculation of the specific impulse
+    of the thuster and if it attains the values we need to achieve the necessary velocity change for orbital decay to initiate atmospheric reentry.
+
+    \textit{Projected Duration: 3 weeks}
+
+\subsubsection{Final Report}
+\label{subsubsec: final report}
+    At the end of this project a final report will be made to report the outcome. It will outline the design process, fabrication process, and testing process.
```

Comment:
>At the end of this project a final report will be made to report the outcome.

This is redundant. It can be reworded
>The project will conclude with the production of a final report.